File: JavaScript/7kyu - draw me a chessboard.js
function chessBoard(rows, columns) {
  return Array(rows).
    fill(Array(columns).fill("#")).
    map((el, i) => el.map((_, j) =>
      i%2 == 0 ? j%2 == 0 ? "O": "X": j%2 == 0 ? "X" : "O"));
}

File: Clojure/7kyu - draw me a chessboard.clj
(ns chessboard.core)

(defn chess-board [rows columns]
  (vec (map-indexed
         (fn [r _]
           (vec (map-indexed
                  (fn [c _]
                    (if (even? (+ r c)) "O" "X"))
                  (range columns))))
         (range rows))))
